### PR TITLE
feat: Visualisation of albums screen

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves [MOPLAT-285]

### Description

image

### PR Checklist

- [x] I tested my changes on **iOS** and **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-285]: https://artsyproduct.atlassian.net/browse/MOPLAT-285?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by pvinis — APPROVED** [review]
I made a comment for the styling, I think it makes more sense to keep it all in one place.

if you agree, change and merge. if you don't agree, merge.

🙌 nice work

File: src/app/screens/HomeTabs/Albums/Albums.tsx [hunk 1]
```diff
@@ -17,18 +19,22 @@ export const Albums = () => (
...
-            <Flex key={album} mb={3} mt={1} backgroundColor="orange">
-              {/* {album.artworkIds.map((artworkId) => (
-              <AlbumListImage slug={artworkId} key={artworkId} />
-            ))} */}
-              <AlbumListImage slug={album.artworkIds[0]} key={album.artworkIds[0]} />
+            <Flex key={album.id} mb={3} mt={1}>
+              <Flex flexDirection="row-reverse" alignItems="flex-end">
+                {album.artworkIds
+                  .slice(0, 3)
+                  .reverse()
```
[the remark this answers is not in the document]
Comment (rajsam003) [reply src/app/screens/HomeTabs/Albums/Albums.tsx:33]: Yes. Thats correct 🤓

File: src/app/screens/HomeTabs/Albums/Albums.tsx [hunk 2]
```diff
@@ -17,18 +19,22 @@ export const Albums = () => (
...
+                  .map((artworkId) => (
+                    <AlbumListImage slug={artworkId} key={artworkId} />
```
[the remark this answers is not in the document]
Comment (rajsam003) [reply src/app/screens/HomeTabs/Albums/Albums.tsx:35]: Then in that case we have to import types for styles, which is too much work. haha

[the remark this answers is not in the document]
Comment (rajsam003) [reply src/app/screens/HomeTabs/Albums/Albums.tsx:35]: If I have to pass it from the parent then I have to create another component to use `useLazyLoadQuery` and get the aspectRatio and pass it to the child component through `style` prop

**Pull request merged**